phylogeny/prototypeSelection.py

```diff
@@ -511,7 +519,9 @@ def prototype_selection_constructive_maxdist(dm: DistanceMatrix,
     # the element to remove first is the one that has smallest distance to all
     # other. "Removing" works by tagging its distance-sum as infinity. Plus, we
     # decrease the number of available elements by one.
-    minElmIdx = currDists.argmin()
+    minElmIdx = next(x for x in currDists.argsort()
```

Comment:
Cool! I'd like to see the same stats for larger numbers, but I guess that is currently too expensive. The remaining question is: is this difference significant? Keep in mind, that the current function is already heuristic - thus we already made the sacrifice of optimality for speed.